ci/README.md

```diff
@@ -0,0 +1,98 @@
+# VIP Go CI Scripts
+
+This directory is not deployed to your VIP site.
```

Comment:
This comment seems out of place. Feels like we need an intro sentence instead first?